itou/static/js/admin.js

```diff
@@ -24,4 +24,6 @@
       }
     }
   });
+  // Remove bootstrap CSS elements not needed for admin
+  document.querySelectorAll(".input-group-append").forEach(elt => { elt.remove(); });
```

Comment:
Peut-être être plus précis sur ce qu’on enlève, pour éviter de retirer un addon qu’un dev voudrait réellement ajouter.
```suggestion
  // Remove toggle password button from the admin.
  document.querySelectorAll(".input-group-append").forEach(elt => {
    if ("password" in elt.children.item(0).dataset) {
      elt.remove();
    }
  });
```

Reply:
Aut-merge is a bitch. Will fix that